src/components/TopBar/TopBar.test.js

```diff
@@ -50,5 +50,74 @@ it('renders', () => {
             container
         );
     });
-    expect(container.textContent).toContain('GridDemotestchildUsername');
+    expect(container.textContent).toContain('GridDemotestchildU');
+});
+
+it('renders', () => {
```

Comment:
instead of duplicating the render test, just test the function that you are trying to test and pass the "John Doe", "John Jack Joe" and "" arguments